docs/source/algorithms/PolarizationCorrectionFredrikze-v1.rst

```diff
@@ -9,28 +13,43 @@
 Description
 -----------
 
-
 Performs wavelength polarization correction on a TOF reflectometer spectrometer.
 
-Algorithm is based on the paper Fredrikze, H, et al. "Calibration of a polarized neutron reflectometer" Physica B 297 (2001).
+The algorithm is based on the paper Fredrikze, H, et al. "Calibration of a polarized neutron reflectometer" Physica B 297 (2001).
```

Comment:
Could we add an explicit reference here? (The doc for `DepolarizedAnalyserTransmission.rst` has an example of how to do this.